firestore/collgroupref.go

```diff
@@ -36,3 +45,107 @@ func newCollectionGroupRef(c *Client, dbPath, collectionID string) *CollectionGr
 		},
 	}
 }
+
+// GetPartitions returns a slice of QueryPartition objects. The number must be
```

Comment:
This is kind of unclear to me. What is a QueryPartition? Why would I want to get it? The type docs don't really illuminate further. Adding more color and a link to the public docs for the feature would help.

Also, "The number" == `partitionCount`?